the-content-guide/readme.md

```diff
@@ -1,20 +1,129 @@
+> **Goal of this guide:** Helping you create the best content you can make.
 
 # How to create awesome content
 
 This guide is intended for external communication but can also be a source for our in-house content developers.
-The target audience are communication professionals. See it as a comprehensive guide, a white paper if you will, to show others our know-how. Because the product that we sell are blog posts or web articles, we should be able to display our skills.
+The target audience is communication professionals. See it as a comprehensive guide, a white paper if you will, to show others our know-how. Because the products that we sell are blog posts and web articles, we should be able to display our skills.
 
 <img src="/images/contentcat.jpg" width="150">
 
-## File storage
+## Intro
 
-Each chapter should be it's own file. When you add a chapter, please provide the name of the file with a number. This number represents the order of the document chapters. After the number comes the chapters' name.
+Why do you need content marketing or branded content? Content marketing is a way to create a bond between a reader and a company. By creating content the reader will thoroughly enjoy, the reader will associate the brand with an awesome feeling and automatically think about that company first whenever searching for a product/service in that domain.
 
-## Pull requests
+Example: A solar cell company creates blogposts about:
+* how a solar cell works  
+* a comparison between green energy products
+* where is the future of solar cell power heading?
+* a photo collage of employees risking their lives on roofs
 
-Make your own branch. When working on the document, create a pull request for each document.
+So how do you attract readers and keep them engaged? There is an increasing demand for *'high quality content'*. But what is high quality content? What ingredients make a nice meal that people will remember? Better yet, how do you get people to ask for and share the recipe?
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, this guide will tell you how to create content you can be proud of!
 
-To know what makes good content — As a communication professional — I can read a comprehensive guide about content creation.
+## Originality & authenticity
+
+Good content is original. [Content Farms](https://en.wikipedia.org/wiki/Content_farm) are no longer wanted in these times. Search engines, such as Google, are getting better and better at detecting these 'clickbait sites' and these sites are not received well. Original content is sure to be shared more often. Moreover, why would you say something when it's not worth saying?
```

Comment:
'something when it's not worth saying'-->'something that's not worth saying' is more natural because 'that' refers to 'something'.